src/BloomBrowserUI/bookEdit/accordion/DecodableRT/DecodableRT.jade

```diff
@@ -36,7 +36,7 @@ html
 			.tableHolder.clear
 				.wordList#wordList
 			#make-letter-word-list-div.clear
-				a#make-letter-word-list(href='javascript:makeLetterWordList();', data-i18n='DecodableReaderTool.MakeLetterWordList') Open a Letter and Word List File
+				a#make-letter-word-list(href='javascript:makeLetterWordList();', data-i18n='DecodableReaderTool.MakeLetterWordReport') Generate a letter and word list report
```

Comment:
It is, that's what `data-i18n='DecodableReaderTool.MakeLetterWordReport'` is for.